install/prometheus-agent-mode/install.yml

```diff
@@ -0,0 +1,13 @@
+id: setup-prometheus-agent-mode
+name: Prometheus Agent Mode
+title: Prometheus Agent Mode
+description: Use the Prometheus Agent to scrape Prometheus endpoints in your Kubernetes cluster and send metrics to New Relic using Prometheus remote write.
+
+target:
+  type: integration
+  destination: kubernetes
+
+install:
+  mode: link
+  destination:
+    url: https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent/
```

Comment:
This doc link is an empty page, we'll just need that to be live before merging

Reply:
Understood. Next week we should meet in order to define the deployment plan.
But the order should be:

* Docs deployed
* Merging this PR
* Merging Quickstarts PRs (7 new integrations + Prometheus Agent) which are referencing the install plan of this PR

cc @seemantk @bpschmitt @thezackm